### ISSUE TITLE
Running command on bash, still getting incompatibility errors

### ISSUE THREAD

**Issue by sfarkas1988** [myspaghetti/macos-virtualbox#238]
Hey, eventhough I call the script with bash, I'm getting the following errors.
I installed cygwin and installed all dependencies while installation.

```
ash ./macos-guest-virtualbox.sh
./macos-guest-virtualbox.sh: line 6: $'\r': command not found
./macos-guest-virtualbox.sh: line 14: $'\r': command not found
./macos-guest-virtualbox.sh: line 15: syntax error near unexpected token `$'{\r''
'/macos-guest-virtualbox.sh: line 15: `function set_variables() {
```

Executing the command in PowerShell opens a bin/bash prompt but after pressing enter it simply closes without any error message.

**Comment by myspaghetti** [comment]
Are you running it in `ash` or `bash`? Your post says `ash`.

**Comment by JamesBeach** [comment]
This error occurs when git automatically gives the file Windows-style CRLF line endings. The repo could probably benefit from having .gitattributes set up to prevent this.

**Comment by myspaghetti** [comment]
@JamesBeach whether a user sets their local end-of-line LF or CRLF should be up to them. Either way I've set the repository's `autocrlf` to `false`.

**Comment by sfarkas1988** [comment]
@myspaghetti it's bash. Forgot to paste the first letter.

**Comment by myspaghetti** [comment]
@sfarkas1988 is Bash configured for LF line-endings while the file has CRLF line-endings as JamesBeach suggested?

**Comment by sfarkas1988** [comment]
I think I kept the default for powershell.
How can I check the easiest?

**Comment by myspaghetti** [comment]
`(Get-Content input.txt -Raw) -match "\r\n$"`

**Comment by sfarkas1988** [comment]
`$ (Get-Content input.txt -Raw) -match "\r\n$"
-bash: syntax error near unexpected token `Get-Content'`

**Comment by myspaghetti** [comment]
That was a PowerShell command and `input.txt` was a sample filename.
On bash:
`grep -lU $'\r$' macos-guest-virtualbox.sh`

**Comment by myspaghetti** [comment]
I can reproduce the issue by replacing LF with CRLF. I'm marking this a duplicate of #28 and closing as `won't fix`

**Comment by sfarkas1988** [comment]
@myspaghetti at least mentioning the issue in the readme would be amazing.

**Comment by eganclose** [comment]
Apologies, but is this is resolved?, if yes can you please advise how ?

**Comment by myspaghetti** [comment]
Yes, this is resolved as `duplicate` of #28 which is resolved as `won't fix`. [You probably set `git` to convert `LF` to `CRLF`](https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings). While I can [configure the repository to override local Git configuration](https://help.github.com/en/github/using-git/configuring-git-to-handle-line-endings#per-repository-settings) I feel like respecting local settings is the correct solution.
